content/docs/cli.md

```diff
@@ -21,18 +21,26 @@ brew tap kudobuilder/tap
 brew install kudo-cli
 ```
 
-or you can compile and install the plugin from your `$GOPATH/src/github.com/kudobuilder/kudo` root folder via:
+Another alternative is `krew` the package manager for kubectl plugins [doc](https://github.com/kubernetes-sigs/krew)
 
 ```bash
-make cli-install
+kubectl krew install kudo
 ```
 
-Another alternative is `krew` the package manager for kubectl plugins [doc](https://github.com/kubernetes-sigs/krew)
+or you can download the CLI binaries from the release page at https://github.com/kudobuilder/kudo/releases/latest and download the release for your platform and OS.  Make executable and add to your path:
 
 ```bash
-kubectl krew install kudo
+export VERSION=0.12.0
+export OS=$(uname | tr '[:upper:]' '[:lower:]')
+export ARCH=$(uname -m)
```

Comment:
```suggestion
VERSION=0.12.0
OS=$(uname | tr '[:upper:]' '[:lower:]')
ARCH=$(uname -m)
```

Reply:
I tend to follow the rule that if an variable does not need to be exported, is should not. Here it's only interpreted by the current shell.